tidy3d/components/heat_charge/monitor.py

```diff
@@ -46,5 +46,32 @@ class VoltageMonitor(HeatChargeMonitor):
     """Electric potential monitor."""
 
 
+class ChargeSimulationMonitor(HeatChargeMonitor):
+    """Charge monitor for Charge simulations."""
+
+    # NOTE: supporting unstructured only for the time being
+    unstructured = True
+
+
+class TemporalTemperatureMonitor(HeatChargeMonitor):
+    """Monitor for time-varying temperature field."""
+
+    # NOTE: supporting unstructured only for the time being
+    unstructured = True
+
+
+class TemporalVoltageMonitor(HeatChargeMonitor):
+    """Monitor for time-varying voltage field."""
+
```

Comment:
Maybe I'm curious what this means exactly? It is in the same format as a FDTD `FieldTimeMonitor`? Mainly out of curiosity?

Reply:
Like maybe it should follow this naming convention? https://github.com/flexcompute/tidy3d/blob/efdc698eddc61bc7d43a1020699be9ed2328a8cc/tidy3d/components/data/monitor_data.py#L29C5-L29C21

Reply:
Actually, I think this shouldn't be here. Both `TemporalTemperatureMonitor` and `TemporalVoltageMonitor` where introduced for transient heat and Electro-Quasi-Static simulations. So I'll remove these from this PR. They should be introduced in a separate PR

Reply:
Nice sounds good!